manage.py

```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/env python
 import os
 import sys
+import warnings
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "dashboard.settings")
 
+    # Regex for which modules to ignore warnings from
```

Comment:
does this mean any warning from djangosaml2 will all be ignored?

Reply:
Yeah, that should probably target the line specifically, let me look into that.

Reply:
I modified this so it only ignores DeprecationWarning(s). I could target line number but that might change if the code changes.

Reply:
looks good!